src/WebApp/appsettings.json

```diff
@@ -1,23 +1,23 @@
 {
-  "ConnectionStrings": {
-    "DefaultConnection": "--placeholder--"
-  },
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
+    "ConnectionStrings": {
+        "DefaultConnection": "Host=localhost;Username=codidact-core;Database=codidact-core"
+    },
+    "Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft": "Warning",
+            "Microsoft.Hosting.Lifetime": "Information"
+        }
+    },
+    "AllowedHosts": "*",
+    "Identity": {
+        "Authority": "http://localhost:8001",
+        "RequireHttpsMetadata": false,
+        "ClientId": "codidact_client",
+        "ClientSecret": "foo",
```

Comment:
Here and in line 3, this PR changed a `~~placeholder~~` value to a concrete value. Is that intentional.

Reply:
This is a relic of me working on rewriting the documentation. I'll put it back for now.

Reply:
This was intentional in the authentication project I added the fitting hash of "foo". With a default value it just works out of the box without changes.

[/docs/defaults/src/WebApp/appsettings.json][1]

  [1]: https://github.com/asynts/codidact-authentication/blob/b16dee46e5e1cf5b1db4ad6868d32d3251576c76/docs/defaults/src/WebApp/appsettings.json#L14